app/grandchallenge/components/templates/components/componentinterface_list.html

```diff
@@ -77,6 +82,10 @@ <h2>{{ object_type|title }} {{ list_type|title }} Options</h2>
                                 <code>/output/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.tif</code>
                             {% else %}
                                 <code>/output/{{ interface.relative_path }}</code>
+                                {% if interface.example_value %}
+                                    <br>
+                                    <a download="json_schema_example.json" href="data:application/octet-stream;charset=utf-8;base64,{{ interface.example_value | b64encode_json }}">Download schema example</a>
+                                {% endif %}
```

Comment:
The example value doesn't need to vary with input and output, so this code block should not be duplicated.

Reply:
I rewrote the part of the view eliminating duplicate code